tools.yml

```diff
@@ -1,5 +1,5 @@
 # https://github.com/chaoss/grimoirelab-perceval
-perceval: ~/perceval/bin/perceval
+perceval: ~/Users/rubenjacobo/anaconda3/bin/perceval
```

Comment:
You should not be editing the tools.yml. Do not use `git add .` to add everything you change. Carefully use `git diff file` to see the changes, then `git add filename` one at a time.